WcaOnRails/spec/models/user_spec.rb

```diff
@@ -180,11 +179,6 @@
       expect(user).to be_valid
     end
 
-    it "does not allow assigning a birthdateless WCA ID to a user" do
```

Comment:
Deleting test without a replacement, see similar comment above

Reply:
Did the same as for the other comment ;)